src/components/ViewList.js

```diff
@@ -114,67 +136,157 @@ const ViewList = () => {
   }, [filterValue]);
 
   return (
-    <div>
-      <h2>View List</h2>
-      <label htmlFor="search">Type to Search</label>
-      <input
-        type="search"
-        name="search"
-        id="search"
-        value={filterValue}
-        onChange={handleSearchChange}
-      />
-      <label htmlFor="Clear" aria-label="Clear search bar"></label>
-      <button onClick={handleClearClick}>Clear</button>
-
-      <ul>
-        {currentList.userList.length > 0 ? (
-          filteredList.map((element, index) => (
-            <li
-              style={{ margin: '10px', padding: '5px' }}
-              key={element.id}
-              className={element.textEstimate}
-              aria-label={`${element.itemName} needs to be purchased ${element.textEstimate}`}
-            >
-              <input
-                type="checkbox"
-                name={element.itemName}
-                id={element.id}
-                value={element.itemName}
-                className="purchased"
-                onChange={handleCheck}
-                checked={element.isPurchased}
-              ></input>{' '}
-              {element.itemName}
-
-              <Button
-                onClick={handleDelete}
-                id={element.id}
-                fontSize="l"
-                padding="10px"
-                margin="0px 5px"
-                borderRadius="10px"
-              >
-                Delete
-              </Button>
-              <Button
-                fontSize="l"
-                padding="10px"
-                margin="0px 5px"
-                borderRadius="10px"
-              >
-                <Link to={`/item/${element.id}`}>Details</Link>
-              </Button>
-            </li>
-          ))
-        ) : (
-          <li>
-            <p> You don't have any items</p>
-            <Link to="/add-items">Add your first item!</Link>
-          </li>
-        )}
-      </ul>
-    </div>
+    <Box bg="brand.600">
+      <Box textStyle="roundedCorners">
+        <Text as="h2" textStyle="h2">
+          List Name
+        </Text>
+        <VisuallyHidden>
+          <FormLabel htmlFor="search">Search</FormLabel>
+        </VisuallyHidden>
+        <Input
+          bgImage="linear-gradient(to right, #00A3C4, #76E4F7)"
+          css={{
+            '::placeholder': {
+              color: 'white',
+            },
+          }}
+          bg="red"
+          w="30%"
+          borderRadius="20px"
+          border="red"
+          type="search"
+          name="search"
+          id="search"
+          placeholder="Search Your List"
+          aria-label="Search your list"
+          value={filterValue}
+          onChange={handleSearchChange}
+        />
+
+        <IconButton
+          colorScheme="blue"
+          aria-label="clear search bar"
+          icon={<CloseIcon />}
+          onClick={handleClearClick}
+        />
+
+        <UnorderedList listStyleType="none">
+          <SimpleGrid columns={3}>
```

Comment:
This violates the HTML rule of the li items being only children of OL or UL. It is currently a child of a div. Maybe swapping SimpleGrid with unorderedList could address this.

Reply:
Try running this chrome extension for accessibility testing: https://accessibilityinsights.io/docs/en/web/overview/ 
There are 2 other errors about having duplicate id's and some elements missing a label (I think the buttons for delete and details)